lnwallet/channel.go

```diff
@@ -9539,15 +9538,12 @@ func (lc *LightningChannel) IdealCommitFeeRate(netFeeRate, minRelayFeeRate,
 		maxFeeRate > maxAnchorCommitFeeRate {
 	case true:
 		commitFeeRate = chainfee.SatPerKWeight(
-			math.Min(
-				float64(netFeeRate),
-				float64(maxAnchorCommitFeeRate),
-			),
+			min(netFeeRate, maxAnchorCommitFeeRate),
```

Comment:
nit: can remove the `chainfee.SatPerKWeight()` cast, as both units are already of that type. Here and further below.